python/llm/example/GPU/QLoRA-FineTuning/alpaca-qlora/alpaca_qlora_finetuning.py

```diff
@@ -310,7 +335,7 @@ def generate_and_tokenize_prompt(data_point):
             eval_steps=100 if val_set_size > 0 else None,
             save_steps=100,
             output_dir=output_dir,
-            save_total_limit=100,
+            save_total_limit=100 if relora_steps < 1 else 4,
```

Comment:
why relora use 4 here?

Reply:
relora will save the whole model (instead of adapter files), too many saves will use large disk space.

Reply:
maybe add a comment here to better remind users of such model size change : )

Reply:
Updated